src/Simulation/CsharpGeneration/Microsoft.Quantum.CsharpGeneration.fsproj

```diff
@@ -21,7 +21,7 @@
 
   <ItemGroup>
     <PackageReference Update="FSharp.Core" Version="4.7.0" />
-    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.11.2006.602-beta" />
+    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.11.2006.1806-alpha" />
```

Comment:
Changing from a beta version to an alpha version means your operating off of not released nugets right? Isn't that going to be an issue?

Reply:
Both -alpha and -beta packages are published to the qdk-alpha NuGet feed. The -alpha packages come from individual repo builds and the -beta ones come from full cross-repo Solid builds.